pkg/base1/test-echo.js

```diff
@@ -79,10 +79,18 @@ QUnit.test("binary", function (assert) {
     channel.send(buffer);
 });
 
-// This is implemented in the C bridge, but not in Python.
-// Nobody uses it, so skip the test for now.
-QUnit.test.skip("fence", function (assert) {
+QUnit.test("fence", async assert => {
     const done = assert.async();
+
+    const response = await fetch(`http://${window.location.hostname}:${window.location.port}/mock/info`);
```

Comment:
Could we integrate that into `pkg/lib/qunit-tests.js` somehow?

Reply:
Indeed in the http channel PR I [have a helper function](https://github.com/martinpitt/cockpit/commit/85f54d839bd22b36de7bfc4d725494a22401603e#diff-8f24a1fc445b04e36587c1207a30cec18e7973665db3543ef9e8854530758e95R18) already. Moving to qunit-tests.js sounds good, I'll experiment with that there, do a rebase, and centralize it.

Reply:
Done in https://github.com/cockpit-project/cockpit/pull/17631/commits/ab40c130da35a517076e0adefafcccf607c158e0